README.md

```diff
@@ -77,22 +94,114 @@ In the sub-folders for plots that intersected with the orthomosaic, there will b
+```bash
+cp /IMG/* "${PWD}/inputs/"
+```
+
+Step 3 copies the optional shapefile files into the same folder as the drone images:
+```bash
+cp /plot_shapes.* "${PWD}/inputs/"
+``` 
+
+In step 4 we copy the experiment.yaml file into the same folder as the drone images:
+```bash
+cp experiment.yaml "${PWD}/inputs/"
+``` 
+
+In step 5 we copy the source files onto the input named volume:
+```bash
+docker run --rm -v "${PWD}/inputs:/sources" -v my_input:/input --entrypoint bash agdrone/canopycover-workflow:latest -c 'cp /sources/* /input/'
+``` 
+
+In step 6 we run the workflow to generate the orothomosaic image using ODM (OrthoDroneMap) and calculate plot-level canopy cover:
+```bash
+docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "${PWD}/inputs:/scif/data"/odm_workflow/images -v my_output:/output -e INPUT_VOLUME=my_input -e OUTPUT_VOLUME=my_output -e "INPUT_IMAGE_FOLDER=/images" -e "OUTPUT_FOLDER=/output" agdrone/canopycover-workflow:latest run odm_workflow plot_shapes.shp my_input my_output
+```
+and we wait until it's finished.
+
+In step 7 we copy the results off the named output volume to our local folder:
+```bash
+docker run --rm -v "${PWD}/outputs:/results" -v my_output:/output --entrypoint bash agdrone/canopycover-workflow:latest -c 'cp -r /output/* /results/'
+```
+The results of the processing are now in the `${PWD}/outputs` folder.
+
+Finally, in step 8 we clean up the named volumes by deleting everything on them:
+```bash
+docker run --rm -v my_input:/input -v my_output:/output --entrypoint bash agdrone/canopycover-workflow:latest -c 'rm -r /input/* && rm -r /output/*'
+```
+
 ### Clean
+
 By executing the [scif](#scif) app named `clean` it's possible to clean up the output folder and other generated files.
 It's recommended, but not necessary, to run the clean app between processing runs by either running this command or through other means.
 
 **Example:**
 
-This docker command line will clean up the output files generated using the [Canopy Cover: Orthomosaic and Shapefile](#can_om_shp) example above.
+This docker command line will clean up the output files generated using the [Canopy Cover: Orthomosaic and Shapefile](#om_can_shp) example above.
 ```bash
-docker run --rm -v ~/inputs:/scif/data/odm/images -v ~/outputs:/scif/data/soilmask agdrone/canopycover-shape-workflow:latest run clean
+docker run --rm -v "${PWD}/inputs:/scif/data/odm_workflow/images" -v "${PWD}/outputs:/scif/data"/soilmask agdrone/canopycover-workflow:latest run clean
 ```
 
 ## Build the container
+
 This section describes how the Docker container could be built.
 Please refer to the [Docker](https://www.docker.com/) documentation for more information on building Docker containers.
 
 ```bash
 cp jx-args.json.example jx-args.json
-docker build --progress=plain -t agdrone/canopycover-shape-workflow:latest .
+docker build --progress=plain -t agdrone/canopycover-workflow:latest .
 ```
+
+## A Note On Docker Sibling Containers
```

Comment:
This section seems to leave more questions than answers - what are the actual security risks? under what conditions should I be concerned with the risk? are there other options? how might I mitigate this risk?

I suspect plenty of scientists would ignore this while sysadmins might consider this a deal-breaker

Reply:
for investigating alternatives: https://github.com/AgPipeline/issues-and-projects/issues/240

Reply:
But can you answer some of the questions above to reassure people / reframe it as in 'under most use cases anticipated for an alpha release - e.g. on desktops and in virtual machines - this should be OK, but this should not be used as is in a production HPC environment without understanding these risks.

Reply:
Its impossible to cover all the risks since they are very dependent upon the operating environment and what the user does. For the same reason it's not really possible to describe the conditions that someone would need to be concerned with the risk. Perhaps adding something like `If you are unsure of how sibling containers may present a risk for your situation we recommend running OpenDroneMap separately and using the [Canopy Cover: Orthomosaic and plot boundaries](om_can_shp) workflow instead`

Reply:
Is it safe to run this on my laptop?

This section still seems like it puts a lot of responsibility on the user to either educate themselves or not use the workflow. And the fact that we don't tell them until the end may compound the distress for someone who walks through the steps in order before reading through the document.

Reply:
If they are reading through, then they should be reading line 97. We can change 'NOTE' to 'IMPORTANT' if that's better.

Reply:
IMO it's safe to run on your laptop. There will be plenty of System Admins that will feel the same, and feel otherwise. I would imagine the same can be said for almost anyone. "Safe" is a very relative term. None of this addresses various policies, best practices, etc. for any individuals circumstances.

Reply:
I've updated the text